.github/workflows/federated-integ-test.yml

```diff
@@ -9,7 +9,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: Federated integ test
+name: Magma Build, Publish & Test Federated Integration
```

Comment:
Just noticed, might this be rather in the category `FeG`, if it is specific to this component?